src/streamlink/plugins/piaulizaportal.py

```diff
@@ -0,0 +1,76 @@
+"""
+$description Japanese live-streaming and video hosting platform owned by PIA Corporation.
+$url ulizaportal.jp
+$type live, vod
+$account Purchased tickets are required.
+$notes Tickets purchased at "PIA LIVE STREAM" are used for this platform.
+"""
+
+import logging
+import re
+import time
+from urllib.parse import parse_qsl, urlparse
+
+from streamlink.plugin import Plugin, pluginmatcher
+from streamlink.plugin.api import validate
+from streamlink.stream.hls import HLSStream
+
+
+log = logging.getLogger(__name__)
+
+
+@pluginmatcher(re.compile(
+    r"https://ulizaportal\.jp/pages/(?P<id>[\da-f]{8}-[\da-f]{4}-[\da-f]{4}-[\da-f]{4}-[\da-f]{12})",
```

Comment:
```suggestion
    r"https?://ulizaportal\.jp/pages/(?P<id>[\da-f]{8}-[\da-f]{4}-[\da-f]{4}-[\da-f]{4}-[\da-f]{12})",

```

Hi, @bastimeyer , could you please help me make the RegEx a little more generic by committing a quick fix?

Reply:
I don't think it's necessary. `https://` is applied to scheme-less input URLs automatically. The plugin expects URLs to be copied by users with the provided UUIDs (nobody will manually type the UUID), so unless stream URLs get shared with `http://` schemes explicitly (which I don't think they will), this is not needed. Apart from that, nobody can push to master directly, so another PR would be needed anyway.

Reply:
> I don't think it's necessary. ...

OK, I agree.
I found that almost all `pluginmatcher`s match both "http" and "https", so I was wondering if I forgot to do so to comply with any rules.

> Apart from that, nobody can push to master directly, so another PR would be needed anyway.

Got it. It's necessary protection.